manual_flagging/yaml/inputs_initial.yaml

```diff
@@ -1,28 +1,114 @@
-# This file contains relevant inputs for the initial_flagging.py script,
-# which will (ideally) be run one time when we first populate the flagging
-# tables in Athena.
+# TODO: Add overview here
 ---
-stat_groups: [
-  "rolling_window",
-  "township_code",
-  "class"
-]
-
-iso_forest: [
-  "meta_sale_price",
-  "sv_price_per_sqft",
-  "sv_days_since_last_transaction",
-  "sv_cgdr",
-  "sv_sale_dup_counts"
-]
+#
+manual_update: True
 
+# The values in housing_run_type, run_tri, and tri_stat_groups will tell the
+# sales val script which properties will be updated. The housing_run_type list
+# currently supports 'res_single_family', 'res_multi_family', and/or 'condos'.
+# The run_tri van be 1, 2, and/or 3, and the values for the tri_stat_groups
+# can be "current" or "og_mansueto".
+
+# 'tri_stat_groups' specifies which flagging methodology will be used for
+# which tri
+
+# Note: If 'og_mansueto' is selected along with either 'res_single_family' or
+# 'res_multi_family', it will trigger an update for all residential property
+# types ('res_single_family' and 'res_multi_family'). This behavior is
+# intrinsic to the 'og_mansueto' script's functionality.
+
+housing_run_type: ["res_single_family"]
+run_tri: [1]
+
+# tri_stat_groups maps the method to be used for each tri.
+tri_stat_groups:
+  # city
+  1: "current"
+  # north
+  2: "og_mansueto"
+  # south
+  3: "og_mansueto"
+
+stat_groups:
+  current:
+    res_single_family:
+      columns:
```

Comment:
Stat groups is first split into `current` and `og_mansueto` keys, which further split into more supplementary info for flagging in each respective methodology.